### PR TITLE
Stg

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
Pushing the changes to production

### CODE REVIEW

File: docs/whats-new/whats-new.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ This page summarises the new features, improvements and bug fixes completed in e
   [**View all production release notes**](/whats-new/production-release-notes)
 
 **Staging release notes**
```
Comment (yilunyou) [inline docs/whats-new/whats-new.md:13]: need to add 1 for 6 oct production release above?

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -2,10 +2,40 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 28 September 2022
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md:5]: need to duplicate this for production release on 6 oct?

File: docs/whats-new/whats-new.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/whats-new.md:13]: Done, waiting for FE and BE version numbers for prod

File: docs/whats-new/staging-release-notes.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/staging-release-notes.md:5]: Included for the Oct 6th prod release.

File: docs/whats-new/whats-new.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply docs/whats-new/whats-new.md:13]: Updated version numbers as they are the same as the staging environment.

File: docs/whats-new/production-release-notes.md [hunk 3]
```diff
@@ -2,10 +2,25 @@
...
+**New features** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">We send emails to the requestor and the TechPass account holder when a TechPass account's status changes.</summary>
+
+Won't it be great to know the progress of your request to enable, disable or terminate your or others' TechPass account?
+
+We have introduced a new feature to send an email to the requestor and account holder when a TechPass account is enabled, disabled and terminated. The requestor will be aware of the progress of their request at all stages.
+
+</details>
```
Comment (yilunyou) [inline docs/whats-new/production-release-notes.md:18]: this is missing?
<summary style="font-size:20px;font-weight:bold">Fixed incorrect validation rules for public officer's email address.</summary>

**Review by yilunyou — APPROVED** [review]

**Pull request merged**